tests/model_forms/tests.py

```diff
@@ -1701,9 +1701,21 @@ class Meta:
             form.errors['author'],
             ['Select a valid choice. That choice is not one of the available choices.']
         )
+        self.assertFalse(form.has_changed())
 
     def test_disabled_multiplemodelchoicefield(self):
         class ArticleForm(forms.ModelForm):
+            categories = forms.ModelMultipleChoiceField(Category.objects.all(),
```

Comment:
Use hanging indent as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).